yarn.lock

```diff
@@ -2790,7 +2861,16 @@
   resolved "https://registry.yarnpkg.com/@types/json5/-/json5-0.0.29.tgz#ee28707ae94e11d2b827bcbe5270bcea7f3e71ee"
   integrity sha1-7ihweulOEdK4J7y+UnC86n8+ce4=
 
+<<<<<<< HEAD
```

Comment:
Seems like we need to run `yarn install` locally so that these conflicts can be resolved